pyi.Dockerfile

```diff
@@ -237,6 +256,8 @@ COPY --from=async-profiler-builder-glibc /tmp/async-profiler/build/async-profile
 COPY --from=async-profiler-builder-glibc /tmp/async-profiler/build/libasyncProfiler.so gprofiler/resources/java/glibc/libasyncProfiler.so
 COPY --from=async-profiler-builder-musl /tmp/async-profiler/build/libasyncProfiler.so gprofiler/resources/java/musl/libasyncProfiler.so
 COPY --from=async-profiler-builder-glibc /tmp/async-profiler/build/fdtransfer gprofiler/resources/java/fdtransfer
+COPY --from=node-package-builder-musl /tmp/module_build gprofiler/resources/node/module/musl
```

Comment:
```
root@e050c04ca431:/app# find gprofiler/resources/node/
gprofiler/resources/node/
gprofiler/resources/node/module
gprofiler/resources/node/module/musl
gprofiler/resources/node/module/musl/11
gprofiler/resources/node/module/musl/11/linux-perf.js
gprofiler/resources/node/module/musl/11/binding.gyp
gprofiler/resources/node/module/musl/11/test
gprofiler/resources/node/module/musl/11/test/test-linux-perf-start.js
gprofiler/resources/node/module/musl/11/test/test-linux-perf-stop.js
gprofiler/resources/node/module/musl/11/test/test-linux-perf-restart.js
gprofiler/resources/node/module/musl/11/linux-perf.h
gprofiler/resources/node/module/musl/11/test.sh
gprofiler/resources/node/module/musl/11/build
gprofiler/resources/node/module/musl/11/build/Release
gprofiler/resources/node/module/musl/11/build/Release/linux-perf.node
gprofiler/resources/node/module/musl/11/build/Release/.deps
gprofiler/resources/node/module/musl/11/build/Release/.deps/Release
gprofiler/resources/node/module/musl/11/build/Release/.deps/Release/obj.target
gprofiler/resources/node/module/musl/11/build/Release/.deps/Release/obj.target/linux-perf
gprofiler/resources/node/module/musl/11/build/Release/.deps/Release/obj.target/linux-perf/linux-perf.o.d
gprofiler/resources/node/module/musl/11/build/Release/.deps/Release/obj.target/linux-perf/linux-perf-listener.o.d
gprofiler/resources/node/module/musl/11/build/Release/.deps/Release/obj.target/linux-perf.node.d
gprofiler/resources/node/module/musl/11/build/Release/.deps/Release/linux-perf.node.d
gprofiler/resources/node/module/musl/11/build/Release/obj.target
gprofiler/resources/node/module/musl/11/build/Release/obj.target/linux-perf.node
gprofiler/resources/node/module/musl/11/build/Release/obj.target/linux-perf
gprofiler/resources/node/module/musl/11/build/Release/obj.target/linux-perf/linux-perf.o
gprofiler/resources/node/module/musl/11/build/Release/obj.target/linux-perf/linux-perf-listener.o
gprofiler/resources/node/module/musl/11/build/config.gypi
```

Many files get copied inside - copy only the ones actually necessary in runtime?

Reply:
Now script copies only `linux-perf.js` and `linux-perf.node` to `/tmp/module_build` directory, so changes in Dockerfile are not required

Reply:
It would be possible to copy `linux-perf.js` only once and then add proper code to gprofiler, but I don't think it is worth changing it since it is only 113 bytes and we will be saving only 15 blocks of disk (~60KB)

Reply:
Not worth changing indeed.